client/app/shared/modernizr/modernizr.js

```diff
@@ -0,0 +1,1406 @@
+/*!
```

Comment:
@cameronscott137 I couldn't find a bundled up Modernizr on NPM so I just threw this one into `shared/modernizr/`